pkg/unvanquished_src.dpkdir/ui/options_mouvements_cosmetic.rml

```diff
@@ -0,0 +1,51 @@
+<rml>
+	<head>
+		<link type="text/rcss" href="/ui/shared/basics.rcss" />
+		<link type="text/template" href="/ui/shared/window.rml" />
+	</head>
+	<body id="options_mouvements_cosmetic"  template="window" style="width: 45em; margin: 1em;">
```

Comment:
Here “mouvement” is french instead of “movement” in english (same for the file name). :wink: